client/src/components/Image.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import axios from 'axios'
+import { Link } from 'react-router-dom'
+import { AuthConsumer, } from "../providers/AuthProvider";
+
+class Image extends React.Component {
+  state = { image: [] }
```

Comment:
call this images. You aren't wrong but we want that variable name to match the data. It is an array of images so calling it images will make more sense down the road.